docs/3.0.0-beta.x/guides/deployment.md

```diff
@@ -189,7 +189,7 @@ Amazon calls cloud storage services **S3**. You create a **bucket**, which holds
 1. Navigate to the `Amazon S3`. In the top menu, click on `Services` and do a search for `s3`, click on `Scalable storage in the cloud`.
 2. Click on the blue `Create bucket` button:
 
-- Give you bucket a unqiue name, under **Bucket name**, e.g. `my-project-name-images`.
+- Give to your bucket an unique name, under **Bucket name**, e.g. `my-project-name-images`.
```

Comment:
Similar to the a/an discussion above, this should actually read: "Give your bucket a unique name, ..." 

You can read this for an explanation: https://www.quora.com/Which-is-the-correct-grammar-usage-a-unique-or-an-unique